docs/source/overview/profiles_and_parameters.rst

```diff
@@ -621,6 +621,11 @@ The following plugins have support for adding args with following parameter Conf
 	| cachekey.pparam        | cachekey.pparam     | ``-o``                       | ``@pparam=-o``                       |
 	+------------------------+---------------------+------------------------------+--------------------------------------+
 
+In order to support difficult configurations at MID/LAST, a
+:term:`Delivery Service` profile parameter is available with parameters
+``LastRawRemapPre`` and ``LastRawRemapPost``, config file ``remap.config``
+and Value the raw remap lines.  The Value in this parameter will be pre
+or post pended to the end of ``remap.config``.
```

Comment:
New special Parameters should be their own sections like the others - this paragraph appears under the `records.config` ConfigFile section, but these Parameters are not related to Parameters with the ConfigFile `records.config`

Reply:
That's my mistake, it's not under `records.config`, I don't know why I thought it was